### ISSUE TITLE
Teaching loading via Groovy scripting

### ISSUE THREAD

**Issue by tischi** [tischi/i2k-2020-s3-zarr-workshop#14]
@joshmoore (cc @constantinpape)

Here's the current Java code:
``` 
OMEZarrS3Reader reader = new OMEZarrS3Reader( "https://s3.embl.de", "us-west-2", "i2k-2020" );
SpimData em = reader.readSpimData( "em-raw.ome.zarr" );
BdvFunctions.show( em );
```

leading to below result 🥳 

![image](https://user-images.githubusercontent.com/2157566/100260370-9bec3a00-2f49-11eb-8bf4-4dd3028627e0.png)

I think that's actually quite nice and we could show this in the workshop, what do you think?

**Comment by tischi** [comment]
Hi @will-moore, adding you to the thread.

**Comment by joshmoore** [comment]
Very nice!

**Comment by tischi** [comment]
Are you 100% happy with the API? Like the names of the class and so on? I am happy to change it!
```
OMEZarrS3Reader reader = new OMEZarrS3Reader( "https://s3.embl.de", "us-west-2", "i2k-2020" );
SpimData em = reader.readSpimData( "em-raw.ome.zarr" );
```
If possible it would be nice if that could stay like this after we've shown it on I2K.

**Comment by joshmoore** [comment]
I don't think I could do better than that at this point, though I admittedly don't know much about the naming of other readers in BDV space. ("SpimData" is more of a concern more me, but that was already discussed with Tobias long ago in Dresden.)

**Comment by tischi** [comment]
> "SpimData" is more of a concern more me, but that was already discussed with Tobias long ago in Dresden

I agree, see the discussion that I just started in gitter (bdv-core).
I would like to push for and I think I could manage until the workshop.

```
OMEZarrS3Reader reader = new OMEZarrS3Reader( "https://s3.embl.de", "us-west-2", "i2k-2020" );
List< SourceAndConverter > images = reader.read( "em-raw.ome.zarr" );
BdvFunctions.show( images.get( 0 ) );
```

**Comment by joshmoore** [comment]
For future reference: https://gitter.im/bigdataviewer/bigdataviewer-core?at=5fbf551d7f947a4e02dcce04

**Comment by tischi** [comment]
@joshmoore @constantinpape @will-moore

I just had a phone call with @NicoKiaru about this. I work with Nico on [bdv-playground](https://github.com/bigdataviewer/bigdataviewer-playground), where one of the main points in fact is to use `SourceAndConverter` as much as possible, because we think this is the currently best image data model (I would say the equivalent of `ImagePlus`)  in imglib2.

Now, I discussed with Nico and we decided that if we want to show fancy coloured (maybe even interactive) label masks on I2K we should do this via the bdv-playground project. 

In addition, I think we should show `BdvFunctions.show( SourceAndConverter sac )`, because `BdvFunctions.show()` is still the best entry point for beginners?! @NicoKiaru would you agree or would you prefer some bdv-playground based entry point to show a `sac` in BDV?

Now, the issue is, if we want to show this cool stuff during I2K we have to ship versions of bdv-playground, bdv-vistools, bdv-core and maybe a bunch of other libraries that are very recent and not compatible with the current Fiji, e.g. they will break BigStitcher. 

Question is whether we (a) should implement the new stuff, and tell the people that they should download a special Fiji for this workshop in order not to break their old one or (b) just live with `SpimData` and black and white label masks but be compatible with the current Fiji?

Opinions?

**Comment by NicoKiaru** [comment]
> @NicoKiaru would you agree or would you prefer some bdv-playground based entry point to show a sac in BDV?

`BdvFunctions.show` is definitely the way to go for the tutorial. bdv-playground is not standard enough. Let's try to come up with some cool and compatible stuff in the near future.

**Comment by joshmoore** [comment]
I understand little of the above, but it all sounds exciting. 🕺
